UGW: de-refer symbolic links to files (if any) while coping target files to linux dir.

diff --git a/include/quilt.mk b/include/quilt.mk
--- a/include/quilt.mk
+++ b/include/quilt.mk
@@ -91,7 +91,7 @@ endef
 kernel_files=$(foreach fdir,$(GENERIC_FILES_DIR) $(FILES_DIR),$(fdir)/.)
 define Kernel/Patch/Default
 	rm -rf $(PKG_BUILD_DIR)/patches; mkdir -p $(PKG_BUILD_DIR)/patches
-	$(if $(kernel_files),$(CP) $(kernel_files) $(LINUX_DIR)/)
+	$(if $(kernel_files),$(CP) -L $(kernel_files) $(LINUX_DIR)/)
 	find $(LINUX_DIR)/ -name \*.rej -or -name \*.orig | $(XARGS) rm -f
 	$(call PatchDir,$(PKG_BUILD_DIR),$(GENERIC_PATCH_DIR),generic/)
 	$(call PatchDir,$(PKG_BUILD_DIR),$(PATCH_DIR),platform/)
